### PR TITLE
[ecosystem] ui layout tweaks

### PR DESCRIPTION
**Pull request by moonclavedev**
## Motivation

UI fixes for Aptos Incentivized Testnet.

### **Global**

- Integrate `font-smooth` anti-aliasing for improved font rendering 
- Utilize `max-width` for main containers instead of `px` padding
- Change markup for Outline Card component to ensure entire element width is true and does not bleed outside of it's container
- Hide Google reCaptcha badge as this was covering items within footer and integrate copy within flow. See here: https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge.-what-is-allowed
- Add calendar icon for Schedule
- Match brand with exact hex for green including an updated palette with complete family of shades overriding "teal"
- Adjust "Success" Alert color scheme to match branding instead of the blue
- Change `<a>` underline styling and hover state
- Border Radius updated to match aptoslabs.com and [explorer](https://explorer.devnet.aptos.dev/)
- Center text within Footer (mobile only)

### **Index**

- Integrate bullet point arrows for `<li>` as flex container, so that on small screens the text doesn't wrap directly below the icon

### **Form**

- Style changes to form elements for increased readability 
- Black checkmark svg for checkbox
- Add `spellcheck="false"` to text input fields to avoid red squiggly lines 
- Validate Node / Submit container is now `position:sticky` (Medium to Large screens) and will no longer be out of view when user has scrolled the page as they complete the fields, eliminating need to scroll back up the page to find the submit button. Set container full width on small screens.
- Convert `<form>` to flex, and set gap to prevent Validate Node / Submit container from overlapping on Medium screens

Screen Recording 2022-05-19 at 7 51 01 AM

Before hiding Google reCaptcha badge:
image

### CODE REVIEW

File: ecosystem/platform/server/app/views/layouts/_flash.html.erb [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+      <div class="ml-3 text-sm font-medium text-teal-800">
         <%= msg %>
       </div>
-      <button type="button" class="ml-auto -mx-1.5 -my-1.5 bg-blue-100 text-blue-500 rounded-lg focus:ring-2 focus:ring-blue-400 p-1.5 hover:bg-blue-200 inline-flex h-8 w-8 dark:bg-blue-200 dark:text-blue-600 dark:hover:bg-blue-300" data-dismiss-target="#alert-2" aria-label="Close" onclick="this.parentElement.remove()">
+      <button type="button" class="flex items-center justify-center ml-auto -mx-1.5 -my-1.5 bg-transparent text-teal-600 hover:text-teal-800 rounded-lg inline-flex w-8 h-8 p-1.5" data-dismiss-target="#alert-2" aria-label="Close" onclick="this.parentElement.remove()">
         <span class="sr-only">Close</span>
         <%= render IconComponent.new(:close, class: 'w-5 h-5') %>
       </button>
     </div>
-  <% elsif type == "alert" %>
```
Comment (zacharydenton) [inline ecosystem/platform/server/app/views/layouts/_flash.html.erb:12]: Add this line back

**Comment by rustielin** [comment]
Please fix the lint error :) https://app.circleci.com/pipelines/github/aptos-labs/aptos-core/4661/workflows/7c24c3d8-527d-4f38-86ac-aa822c11a2de/jobs/23236

```
[ERROR] [eof-newline] [content ecosystem/platform/server/app/assets/stylesheets/application.tailwind.css]: missing newline at EOF

[ERROR] [trailing-whitespace] [content ecosystem/platform/server/app/views/layouts/_flash.html.erb]: trailing whitespace at line 12

Error: there were lint errors
```

**Pull request merged**